dags/common/parsing/json_extractors.py

```diff
@@ -9,15 +9,17 @@ def __init__(
         json_path,
         required=True,
         extra_function=lambda s: s,
+        default="",
```

Comment:
it's not a good practise to have mutable args in the signature https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/